src/Catty/DataModel/Array/SynchronizedArray.swift

```diff
@@ -21,8 +21,8 @@
  */
 
 public class SynchronizedArray<Element> {
-    private let queue = DispatchQueue(label: "SynchronizedArray", attributes: .concurrent)
-    private var array = [Element]()
+    internal let queue = DispatchQueue(label: "SynchronizedArray", attributes: .concurrent)
```

Comment:
When writing the ticket, I forgot that private variables are only accessible in extensions when defined in the same source file. Changing the scope to `internal` is not a good idea, since they will be available from all other source code files, which we try to avoid. Since this is most important and far more important than having the two separate files, please change the scope back to internal and move the extension to the same file again. Thanks a lot!

Reply:
Shall I remove SynchronizedArrayExtension.swift file?

Reply:
Yes please